test/WatchDetection.test.js

```diff
@@ -72,7 +72,9 @@ describe("WatchDetection", () => {
 						filename: "bundle.js"
 					}
 				});
-				const memfs = (compiler.outputFileSystem = new MemoryFs());
+				const memfs = (compiler.outputFileSystem = createFsFromVolume(
```

Comment:
This test case is broken now. Probably for the same reason as the other test case was broken -> `path` can't be root